docs/modules/ROOT/pages/partials/con_hermetic_troubleshooting.adoc

```diff
@@ -0,0 +1,10 @@
+.Troubleshooting
+
+ifdef::troubleshooting_builds[]
+
+If your build fails, be sure to look at your logs. Here are a couple of ways to find them:
+
+* In {ProductName}, from the *Applications* view, select the application build you want to troubleshoot, then from the resulting *Overview* page, select the *Activity* tab. From there, under *Activity By*, select *Pipeline runs*. From the *Name* column, select the build whose logs you want to check, then from the resulting *Pipeline run details* view, select the *Logs* tab.
+* In {ProductName}, from the *Applications* view, select the application build you want to troubleshoot, then from the resulting *Overview* page, select the *Activity* tab. From there, under *Activity By*, select *Pipeline runs*. From the *Name* column, select the build whose logs you want to check. In the resulting *Pipeline run details* view, click *build-container*. When the right panel opens, select the *Logs* tab to see a partial view of the log for that build.
```

Comment:
I think both paragraphs being almost identical make this read here confusing. If I understood it correctly, the second paragraph only is trying to be more specifc about which log subtab needs to be accessed? Maybe that's not necessary, since a failed pipeline will have the task that failed marked with a red cross icon. In any case, I'd avoid the repetition.

Reply:
i see that. (so longin the forest it's hard now to make out the trees!) i'll tweak, @brunoapimentel. well spotted.